test/test_autograd.py

```diff
@@ -4227,6 +4227,49 @@ def maybe_check_raise(fn, should_raise):
         run_test(grad_mode=False, requires_grad=False, is_view=True,
                  should_raise_tuple=(None, None, None))
 
+    def test_inplace_not_requires_grad(self):
+        class MyFn(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, inp):
+                return inp.view_as(inp)
```

Comment:
I thought the case that couldn't be handled was multiple view outputs.

Reply:
For custom Function it's any number of output views.
For regular (codegened) functions, it only multi-output.